### PR TITLE
CWG Poll 3: P2096R2 Generalized wording for partial specializations

### LINKED ISSUE #4324: [2020-11 CWG Motion 3] P2096R2 Generalized wording for partial specializations
http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2020/p2096r2.html

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #4324
Fixes cplusplus/papers#825

### CODE REVIEW

File: source/templates.tex [hunk 1]
```diff
@@ -3377,11 +3358,11 @@
...
+\rSec3[temp.class.spec.match]{Matching of partial specializations}
 
 \pnum
-When a class template is used in a context that requires an instantiation of
-the class,
+When a template is used in a context that requires an instantiation of
+the template,
 it is necessary to determine whether the instantiation is to be generated
 using the primary template or one of the partial specializations.
 This is done by matching the template arguments of the class template
```
Comment (burblebee) [inline source/templates.tex:3368]: Missed edit: "of the class template" -> "of the template"

File: source/templates.tex [hunk 2]
```diff
@@ -3458,18 +3439,18 @@
 \end{example}
 
 \pnum
-In a type name that refers to a class template specialization, (e.g.,
-\tcode{A<int, int, 1>})
+In a name that refers to a specialization of a class or variable template,
+(e.g., \tcode{A<int, int, 1>})
```
Comment (burblebee) [inline source/templates.tex:3443]: Please move the comma to after the "(e.g. ... )"

**Review by burblebee — CHANGES_REQUESTED** [review]
Just 2 minor oversights...

File: source/templates.tex (hunk 1 above)
Comment (jensmaurer) [reply source/templates.tex:3368]: fixed

File: source/templates.tex (hunk 2 above)
Comment (jensmaurer) [reply source/templates.tex:3443]: fixed

**Comment by tkoeppe** [comment]
@burblebee: could you kindly take another look and see if your comments have been addressed?

**Pull request merged**